### ISSUE TITLE
Research on Screenreader Accessibility

### ISSUE THREAD

**Issue by sarah-letson** [cagov/covid19#6027]
Exploring how to best test user experience for those using screen readers.  

Background: 
In early Jan 2022, this feedback came in for the "On this page" link to the county charts. The comment was tagged with the #county-statewide section: "I was trying to find for Kings County what the situation is and I am totally blind using voiceover on the iPhone and I cannot seem to find it." This encouraged us to investigate how we could do some usability testing on the covid19 site (or other sites).

We first investigated if it was possible to run an intercept on ethnio to specifically target people using screen readers. 

Zakiya
Not really an update, but I was very excited about Sarah’s email about Ethnio and screen reader testing. I’ve been focusing on the design system work and haven’t revisited the charts using screen readers since our last convo. I think getting some feedback on the covid charts from real users would serve us going forward. That’s a much better practice than relying on the assumptions of sighted developers like myself.  I would be very interested in doing the engineering work to make it happen — especially since we’re entering a new phase with the design system.

Sarah 
Happy to sidebar on this as needed. It's still not clear to me if it's actually technically possible to know if someone is using a screen reader, since the help article is generalized.

Zakiya 
Looking a little closer and a little more critically - I think you’re right @Sarah Letson Someone would have to self-identify as a screen reader user in some kind of form in order for us to then target them with Ethnio. That sounds like a bad practice to me but maybe I’m wrong. I think the best we could do is to ask people if they use a screen reader on the Ethnio screener  where they opt in to being a tester.

**Comment by sarah-letson** [comment]
Zakiya and I talked today. Zakiya shared that there is not a way to tell if someone is using a screen reader.

We can pivot this project into determining which resources state staff can use to test sites with screen reader users. We can follow-up with: 
- [ ] DOR contacts
- [ ] CDT Accessibility Researchers

**Comment by mediajunkie** [comment]
Seems something worth understanding to me, though not a Covid project per
se?
-- 
Christian Crumlish
Product Manager | Office of Digital Innovation
Digital Crisis Standard
<https://github.com/cagov/covid19/wiki/Crisis-Standard> - covid19.ca.gov
415-672-5759 | ***@***.***

Follow us:
web <https://alpha.ca.gov/> | blog <https://medium.com/cadotgov> | twitter
<https://twitter.com/cadotgov>

**Comment by zakiya** [comment]
We could move it to odi-engineering but I'm afraid it would get lost there.

**Comment by mediajunkie** [comment]
Ok to keep it here nominally.
-- 
Christian Crumlish
Product Manager | Office of Digital Innovation
Digital Crisis Standard
<https://github.com/cagov/covid19/wiki/Crisis-Standard> - covid19.ca.gov
415-672-5759 | ***@***.***

Follow us:
web <https://alpha.ca.gov/> | blog <https://medium.com/cadotgov> | twitter
<https://twitter.com/cadotgov>

**Comment by zakiya** [comment]
Tiny update. 
- I communicated the original feedback message to one of the CDT Accessibility Engineers. They don't have a formal process for doing a review. Waiting for a response.
- The [CA Department of Rehabilitation](https://www.dor.ca.gov/Home/AB434) offers offers training through CDT. Courses relevant to ODI are [currently full](https://cdt.ca.gov/workforce-development/course-schedule/).

**Comment by zakiya** [comment]
We'll be getting an assessment from CDT on the screen readability of the covid charts on the state dashboard. I'll post here when I get it.

**Comment by zakiya** [comment]
Here is the [Accessibility Evaluation Report for Covid19](https://github.com/cagov/covid19/files/8155566/accessibility-evaluation-report-covid-19-ca-gov.docx). I'll leave it to @mediajunkie and @jbum to see if we want to create issues for any of these.

**Comment by jbum** [comment]
Thanks so much!  Reviewing it today.
